Website/src/rendering/src/components/Speakers/FeaturedSpeakers.tsx

```diff
@@ -30,6 +36,13 @@ const getSpeakerNumberToShow = function (props: FeaturedSpeakersProps) {
 };
 
 const FeaturedSpeakers = (props: FeaturedSpeakersProps): JSX.Element => {
+  const { sitecoreContext } = useSitecoreContext<SitecoreContextValue>();
+
+  const isPageEditing = sitecoreContext.pageState === LayoutServicePageState.Edit;
+  const hasSpeakers = props.fields.data?.item;
```

Comment:
`hasSpeakers` is an object or `null` or `undefined` with that code. We could use `!!props.fields?.data?.item` for it to be a boolean and consume less memory.